lib/schema.js

```diff
@@ -189,8 +189,7 @@ function generateColumn(definition) {
     column.size = parseInt(size[1]);
   }
 
-  if (!definition.column_default && definition.is_nullable.toLowerCase() === 'no') {}
-  else
+  if (definition.column_default || definition.is_nullable.toLowerCase() !== 'no')
     column.defaultsTo = definition.column_default;
```

Comment:
For the sake of this PR (and getting you familiar with updating PRs) I'd like to request you to add curly braces.